go/atreugo/go.mod

```diff
@@ -1,4 +1,5 @@
 module main
 
+go 1.13
```

Comment:
@savsgio why ? I thought this was only mandatory for `1.12`

Reply:
Don't worry, It's added automatically by Go

Reply:
@savsgio better to remove it, because of this could be compiled in an other `go` version (pinning it will add more maintenance stuff to do)

could you remove it ?

Reply:
Done!